package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "browser-monkey",
-  "version": "2.7.0",
+  "version": "2.7.1",
```

Comment:
ah you shouldn't do this, it will get bumped when I make a release

Reply:
Removing...